src/Uno.UWP/Devices/Geolocation/Geolocator.Android.cs

```diff
@@ -1,186 +1,346 @@
-						subscriber.Key.TryInitialize();
-					}
-				}
-				else
-				{
-					BroadcastStatus(PositionStatus.Disabled);
-
-					foreach (var subscriber in _positionChangedSubscriptions)
-					{
-						subscriber.Key.WaitForPermissionFromBackground();
-					}
-				}
-			}
-
-			return status;
-		}
-
-		public void WaitForPermissionFromBackground()
-		{
-			CoreApplication.Resuming -= CoreApplication_Resuming;
-			CoreApplication.Resuming += CoreApplication_Resuming;
-		}
-
-		private void CoreApplication_Resuming(object sender, object e)
-		{
-			CoreDispatcher.Main.RunAsync(
-				priority: CoreDispatcherPriority.Normal,
-				handler: InitializeIfPermissionIsGranted
-			);
-		}
-
-		private async void InitializeIfPermissionIsGranted()
-		{
-			// If the user has granted the location permission while the app was in background, Initialize
-			if (await PermissionsHelper.CheckFineLocationPermission(CancellationToken.None))
-			{
-				TryInitialize();
-				CoreApplication.Resuming -= CoreApplication_Resuming;
-			}
-		}
-
-		private LocationManager InitializeLocationProvider(double desiredAccuracy)
-		{
-			var locationManager = (LocationManager)global::Android.App.Application.Context.GetSystemService(Android.Content.Context.LocationService);
-
-			var criteriaForLocationService = new Criteria
-			{
-				Accuracy = Accuracy.Coarse
-			};
-
-			var acceptableLocationProviders = locationManager.GetProviders(criteriaForLocationService, true);
-
-			if (acceptableLocationProviders.Any())
-			{
-				_locationProvider = acceptableLocationProviders.First();
-			}
-			else
-			{
-				_locationProvider = String.Empty;
-			}
-
-			return locationManager;
-		}
-
-		partial void StartPositionChanged()
-		{
-			_positionChangedSubscriptions.TryAdd(this, 0);
-		}
-
-		partial void StopPositionChanged()
-		{
-			_positionChangedSubscriptions.TryRemove(this, out var _);
-		}
-
-		public void OnLocationChanged(Location location)
-		{
-			BroadcastStatus(PositionStatus.Ready);
-			this._positionChanged?.Invoke(this, new PositionChangedEventArgs(location.ToGeoPosition()));
-		}
-
-		public void OnProviderDisabled(string provider)
-		{
-		}
-
-		public void OnProviderEnabled(string provider)
-		{
-		}
-
-		public void OnStatusChanged(string provider, [GeneratedEnum] Availability status, Bundle extras)
-		{
-		}		
-	}
-
-	static class Extensions
-	{
-		private const uint Wgs84SpatialReferenceId = 4326;
-
+#if __ANDROID__
+#pragma warning disable 67
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Android.Locations;
+using Android.OS;
+using Android.Runtime;
+using Uno.Extensions;
+using Windows.ApplicationModel.Core;
+using Windows.Extensions;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.Foundation.Metadata;
+using Windows.UI.Core;
+
+// empty change - forcing build restart
+

```

Comment:
??

Reply:
Frequently builds should be restarted, and such dummy change is the only way I have to make it happen.